src/Model.php

```diff
@@ -1881,7 +1881,7 @@ public function atomic($f, Persistence $persistence = null)
      *
      * @return Query
      */
-    public function action($mode, $args = [])
+    public function toQuery($mode, $args = [])
```

Comment:
We should keep `action` forwarding to `toQuery` with a deprecation notice

Reply:
Yes this is a must because `action()` method is used in projects quite widely.
Also this PR probably need BC-break label.

Reply:
labeled